packages/design-system/src/components/Table/Table.jsx

```diff
@@ -143,10 +145,7 @@ export class Table extends React.PureComponent {
 
 Table.defaultProps = {
   scrollableNotice: (
-    <Alert
-      className="ds-u-margin-y--1 ds-u-font-size--small ds-u-font-weight--normal"
-      role="status"
-    >
+    <Alert className="ds-u-margin-y--1 ds-u-font-weight--normal" role="status">
```

Comment:
I kind of feel like we can remove these classnames altogether, and just add `font-weight: $font-normal` to `ds-c-alert__text`.

Reply:
not really a huge deal, but its nice to avoid some utilities

Reply:
I'd rather avoid adding `fond-weight: $font-normal` to `ds-c-alert__text` just in case it's a breaking change for products using this class which currently inherits the font-weight.

Reply:
Ok, I'm ok with that too. I don't think it will be a breaking change as the default alert font weight is normal, but I agree its fine to play it safe